collector/collector.go

```diff
@@ -0,0 +1,78 @@
+package collector
+
+import (
+	"bytes"
+	"encoding/base64"
+	"encoding/json"
+	"net"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/containous/traefik/cmd/traefik/anonymize"
+	"github.com/containous/traefik/configuration"
+	"github.com/containous/traefik/log"
+	"github.com/containous/traefik/version"
+	"github.com/mitchellh/hashstructure"
+)
+
+const containousURL = "https://collect.traefik.io/619df80498b60f985d766ce62f912b7c"
```

Comment:
Can we have a more expressive name to indicate what's happening behind this URL (like `collectorURL`), possibly even a GoDoc?